drivers/mmc/mmc.c

```diff
@@ -341,8 +342,9 @@ static int sd_send_op_cond(void)
 		}
 
 		/* ACMD41: SD_SEND_OP_COND */
-		ret = mmc_send_cmd(MMC_ACMD(41), OCR_HCS, MMC_RESPONSE_R3,
-				   &resp_data[0]);
+		ret = mmc_send_cmd(MMC_ACMD(41), OCR_HCS |
+			mmc_dev_info->ocr_voltage, MMC_RESPONSE_R3,
```

Comment:
That mmc_dev_info->ocr_voltage should be initialized somewhere? Could you give an example in your driver code?
With this patch, I fail to boot on SD-Card on STM32MP1 platform.

Reply:
There's an issue in STM32MP1 code (mmc_dev_info struct should be zeromem'ed before using it). I'll propose a patch for this.

Reply:
Hi Yann,
On STM32MP1, without setting the voltage, does the controller gets initialized correctly? 

On the Stratix 10 platform, without sending the voltage, the check "if ((resp_data[0] & OCR_POWERUP) != 0U)" will fail.

Reply:
Yes, it is working. But I agree we should provide the supported voltage values in our driver, as it is done for example in STM32MP1 U-Boot driver.

Reply:
I've pushed a patch to correct the issue I had to boot (#1868).
And pushed a other branch with the required patches to manage ocr_voltage (https://github.com/Yann-lms/arm-trusted-firmware/tree/ocr_voltage).
I'll push the P-R for the last patch when the 2 other patches are merged.